src/Compilers/CSharp/Portable/Binder/PatternSwitchBinder.cs

```diff
@@ -101,6 +101,7 @@ internal override void BindPatternSwitchLabelForInference(CasePatternSwitchLabel
         /// the decision tree.
         /// </summary>
         private ImmutableArray<BoundPatternSwitchSection> BindPatternSwitchSections(
+            SyntaxNode syntax,
```

Comment:
Yes; even this `sytnax` parameter is unneeded (it is available in the property `SwitchSyntax`).